pkg/vsphere/disk/vmdk.go

```diff
@@ -58,7 +64,34 @@ func LockedVMDKFilter(vm *mo.VirtualMachine) bool {
 	return vm.Runtime.PowerState == types.VirtualMachinePowerStatePoweredOn
 }
 
-func IsLockedError(err error) bool {
-	// TODO: add check here for actual error
-	return true
+func IsLockedError(op trace.Operation, err error) bool {
```

Comment:
In the import/export specific case we don't care at all as we _know_ it's a single resource we're trying to add.
From a general use perspective, identifying which disks are locked is a definite improvement.

To keep this particular call simple I'd suggest adding a second call (doesn't have to be part of this PR, but likely needed by @emlin):
`LockedDisks(trace.Operation, error) ([]string)`, or
`LockedDisks(trace.Operation, error) ([]object.DatastorePath)`